src/lib.rs

```diff
@@ -45,11 +40,10 @@ pub struct Menu {
 impl Menu {
     /// Create a menu for this window
     // FIXME: Take menu items as parameters or use builder
-    pub fn new(window: &mut Window) -> Self {
+    pub fn new(window: &mut window::Window) -> Self {
```

Comment:
I don't think not importing `Window` has any benefits here.

A good rule of thumb is to always directly import types and modules, but never functions.

Same comment applies to the rest of this PR for changing type imports.

Reply:
According to the rust book, this is how it should be written. I can find it if you want.

Reply:
Please send the link, I'm doubtful

Reply:
https://doc.rust-lang.org/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#creating-idiomatic-use-paths

Reply:
> On the other hand, when bringing in structs, enums, and other items with use, it’s idiomatic to specify the full path